content/guides/tasks/create.md

```diff
@@ -0,0 +1,10 @@
+---
+rank: 2
+related_endpoints:
+  - post_tasks_id
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Create a task
```

Comment:
I think this is missing content, am I wrong?

Reply:
I thought I committed this up after I wrote it. I'll either see about restoring the file and recommitting or rewriting.